Cargo.lock

```diff
@@ -73,6 +73,15 @@ dependencies = [
  "memchr",
 ]
 
+[[package]]
+name = "android_system_properties"
```

Comment:
Looks suspicious to me. Could you please drop `Cargo.lock`, and run `cargo clean` and `cargo build`?